src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -65,7 +62,9 @@ end
 
     @unpack variable_bounds = limiter.cache.subcell_limiter_coefficients
 
-    var_min = variable_bounds[index]
+    (; variable_bounds) = limiter.cache.subcell_limiter_coefficients
+    (; bounds_order) = limiter.cache
+    var_min = variable_bounds[bounds_order["$(variable)_min"]]
```

Comment:
I don't see why you have to have a vector of arrays, but then index this vector by an index that you get from a dict. Why not use a dict of arrays directly?
```suggestion
    var_min = variable_bounds[Symbol("$(variable)_min")]
```